server/filters/get-main-media.js

```diff
@@ -0,0 +1,14 @@
+export default function getMainMedia(arr) {
```

Comment:
Nice. Might be worth keeping the `getPrincipleMainMedia` as `mainMedia` is already a thing to avoid confusion.